pkg/proxy/util/utils_linux.go

```diff
@@ -0,0 +1,73 @@
+// +build linux
```

Comment:
I would suggest put this file in pkg/proxy/ipvs and rename it to `netlink_unsupported.go`

Reply:
May be `netlink_linux.go` ?

Reply:
true :)